src/server/routes/api-routes.js

```diff
@@ -5,4 +5,24 @@ const router = express.Router();
 
 router.use('/items', items);
 
+router.use('/*', (req, res) => {
+  if (res.statusCode === 200 || res.statusCode === 304) {
+    return;
+  }
+  res.status(404).send('API Request > 404 - Page Not Found');
+});
+
```

Comment:
Is that one at master branch?? 
We just solve that in this PR 
Please check it out again